setup.py

```diff
@@ -56,5 +57,8 @@ def get_version():
         "Development Status :: 3 - Alpha",
         "Programming Language :: Python",
         "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
-    ]
+    ],
+
+    ext_modules = cythonize("src/aspire/utils/BlockDiagonal.pyx")
```

Comment:
Probably we should use low case for the python file name for the consistency with other source files. I like the name such as blk_diag_matrix.py.

Reply:
sure, I will change, np.